modules/biobank/php/container.class.inc

```diff
@@ -0,0 +1,350 @@
+     *
+     * @return none
+     */
+    public function setCenterId(int $centerId) : void
+    {
+        $this->centerId = $centerId;
+    }
+
+    /**
+     * Gets the ID of the container's current centerId
+     *
+     * @return int
+     */
+    public function getCenterID() : int
+    {
+        return $this->centerId;
+    }
+
+    /**
+     * Sets the parent container's ID
+     *
+     * @param ?int $parentContainerId the parent container's ID
+     *
+     * @return void
+     */
+    public function setParentContainerId(?int $parentContainerId = null) : void
+    {
+        $this->parentContainerId = $parentContainerId;
+    }
+
+    /**
+     * Gets the parent container's ID
+     *
+     * @return int|null
+     */
+    public function getParentContainerId() : ?int
+    {
+        return $this->parentContainerId;
+    }
+
+    /**
+     * Sets the IDs of the children containers
+     *
+     * @param ?array $childContainerIds array of IDs of children container
+     *
+     * @return void
+     */
+    public function setChildContainerIds(?array $childContainerIds = null) : void
+    {
+        $this->childContainerIds = $childContainerIds;
+    }
+
+    /**
+     * Gets the IDs of the children containers
+     *
+     * @return ?array
+     */
+    public function getChildContainerIds() : ?array
+    {
+        return $this->childContainerIds;
+    }
+
+    /**
+     * Sets the container's current coordinate in storage
+     *
+     * @param ?int $coordinate the container's current coordinate
+     *
+     * @return void
+     */
+    public function setCoordinate(?int $coordinate = null) : void
+    {
+        $this->coordinate = $coordinate;
+    }
+
+    /**
+     * Gets the container's current coordinate in storage
+     *
+     * @return ?int
+     */
+    public function getCoordinate() : ?int
+    {
+        return $this->coordinate;
+    }
+
+    /**
+     * Gets an array representing the state of the Container
+     *
+     * @return array
+     */
+    public function jsonSerialize() : array
+    {
+        return array(
+                'id'                => $this->id,
+                'barcode'           => $this->barcode,
+                'typeId'            => $this->typeId,
+                'dimensionId'       => $this->dimensionId,
+                'temperature'       => $this->temperature,
+                'statusId'          => $this->statusId,
+                'originId'          => $this->originId,
+                'centerId'          => $this->centerId,
+                'parentContainerId' => $this->parentContainerId,
+                'childContainerIds' => $this->childContainerIds,
+                'coordinate'        => $this->coordinate,
+               );
+    }
+
+    // XXX: This function must be implemented because this clas implements
+    // LORIS/Data/DataInstance. However, it is not currently being used anywhere
+    // and I don't believe it necessary.
```

Comment:
It is not implemented, can we just get rid of it? @ridz1208 @johnsaigle

Reply:
Because this class implements `\DataInstance`, it needs to have a `toJSON` function or else an error will be thrown. Like Rida mentioned in one of his review comments though, it does seem relatively harmless so I will leave it here and remove the comment.